src/main.rs

```diff
@@ -221,7 +233,12 @@ fn notmain() -> Result<i32, anyhow::Error> {
         bail!("no probe was found")
     }
     log::debug!("found {} probes", probes.len());
-    let probe = probes[0].open()?;
+    let probe = if let Some(probe_opt) = opts.probe.as_deref() {
+        let selector: probe_rs::DebugProbeSelector = probe_opt.try_into()?;
+        probe_rs::Probe::open(selector)?
+    } else {
+        probes[0].open()?
```

Comment:
could you change this to throw an error if there's more than one probe? the error should say something like "more than one probe found; use `--probe` to specify which one to use"

Reply:
Yes, good idea.